data/hai/unfettered 0 prologue.txt

```diff
@@ -506,57 +509,1298 @@ mission "Unfettered Jump Drive Trading"
+			choice
+				`	"Of course. What do you need me to do?"`
+					goto action
+				`	"What do you mean, brothers and sisters from Hai space? Are you not different?"`
+					goto brethren
+				`	"Why do you need me to do it?"`
+			label whyme
+			`	"Well, normally they would be able to pay to come here, but the price now is quite high; too high. The Fettered Hai make it difficult for us to go anywhere; as you have probably noticed, they have a fleet on the border between us. And not many Hai captains are willing to come here. If they do, they charge large fees because of the obvious risks involved - for instance, getting killed in the crossfire. Yet you somehow made your way here, probably also under fire from our own people, but, if you choose to assist us, I'll ensure you won't have any issues getting out of here. So, are you willing to help us?"`
+			choice
+				`	"Of course. What do you need me to do?"`
+					goto action
+				`	"This seems like a bad idea. Ask someone else."`
+					decline
+			label brethren
+			`	"Then you have heard about the stories of the Drak modifying our people..." He sighs and deflates a bit. "I do not believe in such things. What makes a Hai - or any other species - 'true' is in his heart. It will take him wherever he wants to go." He pauses, thinking, and when he speaks again it is with deep conviction. "Even if we war with the Elders, it does not mean we hate other Hai, nor do they hate us! We do it only because we must, in order to keep living our way. We can still communicate with our people, and eventually they will all be freed from the limits of the Elders."`
+			choice
+				`	"Will this get me into any trouble with the Hai?"`
+				`	"What limits?"`
+					goto limits
+				`	"Why do you need me to do it anyway?"`
+					goto whyme
+			`	"Quite the contrary! They may not believe in our way of doing things, but they would never dare to restrict the movement of their own people. It would be against all they stand for. Even if we behave differently, they know we are still Hai and that we would never hurt Hai civilians." He seems confident.`
+			choice
+				`	"What is it exactly that you want me to do?"`
+					goto action
+				`	"I believe they would not have their place here. I will not bring them."`
+					decline
+			label limits
+			`	"I mean that they are prevented from reaching their true potential. Once we had many stars and planets to call ours, but since then most of the Hai have decided to stop changing, to stay peaceful with all species, no matter how primitive. They have chosen to decay, to grow soft. The Unfettered are the ones who choose to keep our legacy, our way of life, and then to evolve, to grow."`
```

Comment:
Technically they've chosen to re-imagine/reinvigorate/reinstate a legacy way of life - one they re-learned recently from the Korath raids. 
They're not a continuation of an older thing, they're a reboot.

Reply:
well he just presents it as if the Unfettered were always a thing, resisting the bad moves of the Hai government and such before the raids happened

Reply:
It would be nice if the Player had the opportunity to call him out on it and get that response then. We should have an acknowledgement that their philosophy has justifications for explanations that are otherwise blatantly false on the face of it.

Reply:
True, I'll make it an option if you have done Hai: ask about the Unfettered